### ISSUE TITLE
Upgrading sentry-raven gem to 0.15.5

### ISSUE THREAD

**Issue by giedriusr** [getsentry/raven-ruby#463]
Hello,

Today I've updated our sentry-raven gem from `0.15.3` to `0.15.5`. Whenever an exception happens, instead of pointing the issue it throws an error:

```
Puma caught this error: Unknown interface: 2 (Raven::Error)
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/sentry-raven-0.15.5/lib/raven/event.rb:85:in `interface'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/sentry-raven-0.15.5/lib/raven/event.rb:91:in `[]'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/railties-4.2.5.1/lib/rails/rack/logger.rb:39:in `call_app'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/railties-4.2.5.1/lib/rails/rack/logger.rb:22:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/quiet_assets-1.1.0/lib/quiet_assets.rb:27:in `call_with_quiet_assets'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/newrelic_rpm-3.15.0.314/lib/new_relic/agent/instrumentation/middleware_tracing.rb:96:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/request_store-1.2.0/lib/request_store/middleware.rb:8:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/newrelic_rpm-3.15.0.314/lib/new_relic/agent/instrumentation/middleware_tracing.rb:96:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/actionpack-4.2.5.1/lib/action_dispatch/middleware/request_id.rb:21:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/newrelic_rpm-3.15.0.314/lib/new_relic/agent/instrumentation/middleware_tracing.rb:96:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/rack-1.6.4/lib/rack/methodoverride.rb:22:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/newrelic_rpm-3.15.0.314/lib/new_relic/agent/instrumentation/middleware_tracing.rb:96:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/rack-1.6.4/lib/rack/runtime.rb:18:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/newrelic_rpm-3.15.0.314/lib/new_relic/agent/instrumentation/middleware_tracing.rb:96:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/activesupport-4.2.5.1/lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/newrelic_rpm-3.15.0.314/lib/new_relic/agent/instrumentation/middleware_tracing.rb:96:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/rack-rewrite-1.5.1/lib/rack/rewrite.rb:24:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/newrelic_rpm-3.15.0.314/lib/new_relic/agent/instrumentation/middleware_tracing.rb:96:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/rack-1.6.4/lib/rack/lock.rb:17:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/newrelic_rpm-3.15.0.314/lib/new_relic/agent/instrumentation/middleware_tracing.rb:96:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/actionpack-4.2.5.1/lib/action_dispatch/middleware/static.rb:116:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/newrelic_rpm-3.15.0.314/lib/new_relic/agent/instrumentation/middleware_tracing.rb:96:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/rack-1.6.4/lib/rack/sendfile.rb:113:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/newrelic_rpm-3.15.0.314/lib/new_relic/agent/instrumentation/middleware_tracing.rb:96:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/sentry-raven-0.15.5/lib/raven/integrations/rack.rb:53:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/newrelic_rpm-3.15.0.314/lib/new_relic/agent/instrumentation/middleware_tracing.rb:96:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/railties-4.2.5.1/lib/rails/engine.rb:518:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/railties-4.2.5.1/lib/rails/application.rb:165:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/railties-4.2.5.1/lib/rails/railtie.rb:194:in `public_send'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/railties-4.2.5.1/lib/rails/railtie.rb:194:in `method_missing'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/newrelic_rpm-3.15.0.314/lib/new_relic/agent/instrumentation/middleware_tracing.rb:96:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/rack-1.6.4/lib/rack/content_length.rb:15:in `call'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/puma-2.16.0/lib/puma/server.rb:557:in `handle_request'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/puma-2.16.0/lib/puma/server.rb:404:in `process_client'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/puma-2.16.0/lib/puma/server.rb:270:in `block in run'
/Users/giedrius/.rvm/gems/ruby-2.3.0/gems/puma-2.16.0/lib/puma/thread_pool.rb:106:in `block in spawn_thread'
```

Tried to downgrade to 0.15.3 and it works as expected.
EDIT: `0.15.4` version also works!
Workstation environment:
```
$ ruby -v
ruby 2.3.0p0 (2015-12-25 revision 53290) [x86_64-darwin14]
$ rails -v
Rails 4.2.5.1
$ gem -v
2.5.1
$ puma -v
puma version 2.16.0
```

Anyone else is having same issue? Thank you!

**Comment by tierra** [comment]
I'm seeing the same thing... running inside of a docker container if that matters.

**Comment by eugeneius** [comment]
This looks like the same problem as #460 - a `Raven::Event` object is being returned from the middleware instead of a Rack response. The next release will include a fix: #461

**Comment by giedriusr** [comment]
:+1:

**Comment by nateberkopec** [comment]
Fixed in 0.15.6.

**Comment by giedriusr** [comment]
:+1: awesome
